packages/extension-polkagate/src/fullscreen/sendFund/InputPage.tsx

```
@@ -99,6 +100,14 @@ export default function InputPage ({ address, assetId, balances, inputs, setInpu
   const { api, chain, formatted } = useInfo(address);
   const teleportState = useTeleport(address);
 
+  const isForeignAsset = assetId && assetId.startsWith('0x');
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)